bin/p2-bin2pod/main.go

```diff
@@ -56,7 +56,7 @@ func podId() types.PodID {
 	if *id != "" {
 		return types.PodID(*id)
 	}
-	return types.PodID(path.Base(*executable))
+	return types.PodID(path.Base((*executable).Path))
```

Comment:
I hate to nit but can this be just `executable.Path` since pointers inherit the method sets of their pointees?

Not that I would refuse to merge on the basis of this, but it's a question that makes me curious